### DISCUSSION TITLE
TCAS in Dev Version

### DISCUSSION THREAD

**Discussion by Hor1463** [flybywiresim/aircraft#6642] [General]
Hi, is the TCAS feature now emplemented in the Dev Versionof the FBWA320?
Roland

**Comment by Hor1463** [comment]
I have the last Dev Version, but there is no TCAS Signal from the AI Traffic.
Roland

**Reply by tracernz** [reply]
Only experimental for now.

**Comment by Hor1463** [comment]
Oh, hopethat it will be in the Dev. Version next time.
By the way, how can I change from Dev. to the experimental? Have I uninstall the Dev first?
Roland

**Reply by CptBayer** [reply]
use the installer .... it´s easy :-)

![grafik](https://user-images.githubusercontent.com/76531893/152550834-e2699c95-0b4d-4a0e-8add-948379623eb3.png)

**Comment by Hor1463** [comment]
Ok, thx
